### ISSUE TITLE
File.store_var() not writing data.

### ISSUE THREAD

**Issue by GalanCM** [godotengine/godot#3273]
It appears that File.store_var() isn't working in master. I've used the following code to test it:
```GDScript

extends Node

func _ready():
	var new = File.new()
	if not new.file_exists("user://save_data"):
		new.open("user://save_data", File.WRITE)
		new.store_var('test')
		print('new')
		new.close()
	else:
		new.open("user://save_data", File.READ_WRITE)
		print(new.get_var())
		new.close()
```
I would have thought that I was just screwing something up, but I was using similar code in a game that I made in 1.1 stable last month, and it worked fine, and was working in master as of a few weeks ago. I made a cursory attempt to see if I could figure out what change may have done it, but no luck.

**Comment by GalanCM** [comment]
After some more experimentation, it looks like the problem only occur in File.READ_WRITE mode.

**Comment by reduz** [comment]
Yes, therr was a bug.. Use write instead

**Comment by kubecz3k** [comment]
I can confirm the bug in 79c1392d17156aeb8d6a2abca16db59b1f8312cc build (07.01.2016). Also the bug was not there in 02d6669a3849c55e415f37b23b36b49f23319957 build.

**Comment by reduz** [comment]
restored the old READ_WRITE behavior, and new behavior is now added as WRITE_READ. I didn't realize that many users were using this functionality

**Comment by reduz** [comment]
damn closed the wrong issue, well this is fixed

**Comment by akien-mga** [comment]
Fixed by 52e53d4 indeed.
